File: concepts/java-writing-readable-maintainable-code/src/test/java/dump/CommonUtils.java
package dump;

import java.time.Instant;
import java.time.ZoneId;
import java.util.Random;

public class CommonUtils {


    public static int countOccurrences(String stringToSearch, char charToFind){
        int count = 0;
        for (int i=0; i < stringToSearch.length(); i++){
            if (stringToSearch.charAt(i) == charToFind){
                count++;
            }
        }
        return count;
    }

    public static void printNowNewYorkTime(){
        String now = Instant.now().atZone(ZoneId.of("America/New_York")).toString();
        System.out.println(now);
    }

    public static int generateRandomNumberBetween(int low, int high){
        return new Random().nextInt(high-low) + low;
    }
}

File: software-engineering-concepts/Writing-Readable-Maintainable-Code/PHP/Writing_Readable_and_Maintainable_Code/01Naming/Problem/CommonUtils.php
<?php

class CommonUtils
{
  public static function countOccurrences(string $stringToSearch, $charToFind)
  {
    $count = 0;
    $length = strlen($stringToSearch);
    for ($i = 0; $i < $length; $i++) {
      if ($stringToSearch[$i] == $charToFind) {
        $count++;
      }
    }
    return $count;
  }

  public static function generateRandomNumberBetween(int $low, int $high)
  {
    return rand($high, $low);
  }

  public static function convertCurrency()
  {
    return 0;
  }

  public static function printNowNewYorkTime()
  {
    $now = new DateTime("now");
    $now->setTimezone(new DateTimeZone("America/New_York"));
    echo $now->format("Y-m-d H:i:s");
  }
}
